chasm/engine.go

```diff
@@ -42,8 +42,7 @@ type Engine interface {
 	PollComponent(
```

Comment:
I've removed `operationFn` from `PollComponent` since we don't need poll-with-mutation yet.